code/datums/traits/negative.dm

```diff
@@ -1,5 +1,87 @@
 //predominantly negative traits
 
+/datum/trait/depression
+	name = "Depression"
+	desc = "You sometimes just hate life."
+	mob_trait = TRAIT_DEPRESSION
+	value = -1
+	gain_text = "<span class='danger'>You start feeling depressed.</span>"
+	lose_text = "<span class='notice'>You no longer feel depressed.</span>" //if only it were that easy!
+	medical_record_text = "Patient has a severe mood disorder causing them to experience sudden moments of sadness."
+
+
+
+/datum/trait/family_heirloom
```

Comment:
What we then need next is a possibility for antags to buy a steal heirloom trait which can give them a positive buff.